src/containers/course-section/resources-list/ResourcesList.tsx

```diff
@@ -77,47 +72,71 @@ const ResourcesList: FC<ResourcesListProps> = ({
     [setResources]
   )
 
-  const resourcesList = items.map((item, i) => (
-    <Draggable draggableId={item._id.toString()} index={i} key={item._id}>
-      {(provided: DraggableProvided, snapshot: DraggableStateSnapshot) => (
-        <Box
-          ref={provided.innerRef}
-          sx={styles.section(snapshot.isDragging)}
-          {...provided.draggableProps}
-        >
-          <Box
-            className='dragIcon'
-            sx={styles.dragIcon}
-            {...provided.dragHandleProps}
-          >
-            <DragIndicatorIcon />
-          </Box>
-          <ResourceItem
-            deleteResource={deleteResource}
-            editResource={editResource}
-            resource={item}
-            setResourceAvailability={setResourceAvailability}
-          />
-        </Box>
-      )}
-    </Draggable>
-  ))
+  const handleDragEnd = useCallback(
+    (event: DragEndEvent) => {
+      const { active, over } = event
+      if (active && over) {
+        setResources((prevResources) => {
+          const activeIndex = prevResources.findIndex(
+            (item) => item._id === active.id
+          )
+          const overIndex = prevResources.findIndex(
+            (item) => item._id === over.id
+          )
+          if (activeIndex !== -1 && overIndex !== -1) {
+            return arrayMove(prevResources, activeIndex, overIndex)
+          }
+          return prevResources
+        })
+      }
+    },
+    [setResources]
+  )
+
+  const renderItem = (item: CourseResources, isDragOver = false) => (
+    <SortableWrapper
+      id={item._id}
+      key={item._id}
+      onDragEndStyles={styles.section(isDragOver)}
+      onDragStartStyles={styles.section(true)}
```

Comment:
What does `true` in `styles.section(true)` mean?

Reply:
It's mean, that if state `true`, this piece of styles will be added
```
section: (isDragging: boolean) => ({
    mb: '20px',
    backgroundColor: 'basic.white',
    borderRadius: '6px',
    ...(isDragging && {
      boxShadow: `0px 3px 16px 2px ${palette.primary[300]}`,
      border: `2px solid ${palette.primary[300]}`
    })
  }),
```

Reply:
Makes sense, thanks